src/components/DigitalConverterComponent.cpp

```diff
@@ -0,0 +1,105 @@
+#include <hyro/DigitalConverterComponent.hpp>
+
+namespace hyro
+{
+
+std::shared_ptr<hyro::HyroLogger> DigitalConverterComponent::s_logger = hyro::HyroLoggerManager::CreateLogger("DigitalConverterComponent");
+
+DigitalConverterComponent::DigitalConverterComponent (URI uri): hyro::Component(uri)
+{
+  /** Initialize object here */
+};
+
+hyro::Result DigitalConverterComponent::init (const hyro::ComponentConfiguration & config)
+{
+  /** Dummy output for fixing the DynamicProperty bug for unit tests */
+  std::shared_ptr<ChannelOutput<std::vector<int>>>
+  m_dummy = this->registerOutput<std::vector<int>>("fix_dynamic"_uri, config);
+  
+  /** Setting an output channel for sending the analog signal */
+  m_input  = this->registerInput<Signal>("signals"_uri, config);
+
+  /** Setting an output channel for sending the thresholded signal */
+  m_output = this->registerOutput<float>("digital_signals"_uri, config);
+
+  /** Setting the parameters from configuration */
+  m_amplitude = config.parameters.getParameter<double>("amplitude", 2.0);
```

Comment:
It seems that `m_amplitude` and `m_threshold` are just used here to initialize the `m_thresh`, is that right?
If it's the case, them can be defined locally, not globally.

Reply:
Actually I am also using it inside the get/set functions of my dynamic properties in order to update the `m_thresh.setThreshold(m_amplitude,m_threshold)` method. Should I split this method into two methods: eg. :  `m_thresh.setAmplitude(m_amplitude)`  and `m_thresh.setThreshold(m_threshold)` ?

Reply:
Yeah, I'd go for it and avoid storing the same information twice in different places.

```
bool DigitalConverterComponent::setThreshold(double threshold)
{ 
  return m_thresh.setThreshold(m_threshold);
}


double DigitalConverterComponent::getThreshold()
{ 
  return m_thresh.getThreshold();
}
```

Reply:
:+1: